src/components/publiq-ui/Button.stories.mdx

```diff
@@ -4,10 +4,11 @@ import { Button, ButtonVariants } from './Button.js';
 <Meta title="Components/Button" />
 
 export const commonArgs = {
```

Comment:
Where is the name 'commonArgs' coming from? Aren't these the actual props of the component?

Reply:
You could name name `commonArgs` `commonProps` for example, it's named this way because of the storybook naming of `args`.